templates/d.erb

```diff
@@ -9,6 +9,11 @@ PATH=/usr/local/bin:/usr/local/sbin:/nail/sys/bin:/usr/sbin:/usr/bin:/sbin:/bin
 <% else -%>
 PATH=/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/bin:/usr/local/sbin
 <% end -%>
+<% if ! @env.empty? -%>
+<% @env.each do |variable, value| %>
```

Comment:
order of `.each` over a map is non-deterministic, use this instead:

    <% @env.keys.sort.each do |variable| value=@env[variable] %>